bundles/action/org.openhab.action.pushbullet/src/main/java/org/openhab/action/pushbullet/internal/PushbulletAPIConnector.java

```diff
@@ -126,7 +133,16 @@ private static boolean sendPush(String botName, String recipient, String title,
         push.setTitle(title);
         push.setBody(body);
         push.setType(type);
-        push.setEmail(recipient);
+
+        if (recipient != null) {
```

Comment:
else ?

What if recipient is null? Shouldn't there be a throw in that case?

Reply:
Same reason as above. If the recipient is null, we would fallback to the default behavior.